src/init.cpp

```diff
@@ -512,7 +510,10 @@ std::string HelpMessage(HelpMessageMode mode)
     if (showDebug) {
         strUsage += HelpMessageOpt("-fedpegscript=<hex>", _("Change federated peg to use a different script.") +
             " " + _("This creates a new chain with a different genesis block."));
+        strUsage += HelpMessageOpt("-signblockscript=<hex>", _("Change chain to be signed and validated with a different script.") +
```

Comment:
Perhaps -signblockscript and -initialfreecoins could be moved to  https://github.com/ElementsProject/elements/blob/elements-0.14.1/src/chainparamsbase.cpp#L16 ?

Reply:
My gut feeling is that these are even more advanced than chain-name and should remain hidden.

Reply:
sure, they can remain under showDebug, I'm just proposing to move them from init to chainparamsbase, to the chain selection section.